File: qrcode/ios/Classes/QrcodePlugin.m
#import "QrcodePlugin.h"
#import "QRCaptureViewFactory.h"

@implementation QrcodePlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    QRCaptureViewFactory *factory = [[QRCaptureViewFactory alloc] initWithRegistrar:registrar];
    [registrar registerViewFactory:factory withId:@"plugins/qrcode_flutter_view"];
}

@end

File: qrcode/android/src/main/kotlin/com/example/qrcode/QrcodePlugin.kt
package com.example.qrcode
import io.flutter.plugin.common.PluginRegistry.Registrar

class QrcodePlugin {
  companion object {
    @JvmStatic
    fun registerWith(registrar: Registrar) {
      registrar.platformViewRegistry().registerViewFactory("plugins/qrcode_flutter_view", QRCaptureViewFactory(registrar))
    }
  }
}
